src/services/netflix/NetflixApi.ts

```diff
@@ -444,19 +444,7 @@ class _NetflixApi extends ServiceApi {
 	}
 
 	getSession(): Promise<NetflixSession | null> {
-		return ScriptInjector.inject<NetflixSession>(this.id, 'session', '', () => {
-			let session: NetflixSession | null = null;
-			const { netflix } = window;
-			if (netflix) {
-				const { userInfo } = netflix.reactContext.models;
-				const authUrl = userInfo.data.authURL;
-				const profileName = userInfo.data.name;
-				if (authUrl) {
-					session = { authUrl, profileName };
-				}
-			}
-			return session;
-		});
+		return ScriptInjector.inject<NetflixSession>(this.id, 'session', '');
```

Comment:
I've pinpointed that this is the point of failure.

Reply:
@MrMamen I've pushed some changes, can you see if it's fixed now?